src/Commands/Pages/SetPage.cs

```diff
@@ -64,6 +64,12 @@ public class SetPage : PnPWebCmdlet, IDynamicParameters
         [Parameter(Mandatory = false)]
         public bool ShowPublishDate;
 
+        [Parameter(Mandatory = false)]
+        public SwitchParameter Like = false;
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter UnLike = false;
```

Comment:
I think we can use a single parameter for this ? Just keep the parameter `Like` and in the code we check if its present. If the value of it is true , we like the page , if false, we unlike.

So end users will have to do something like 

```
Set-PNPPage "123.aspx" -Like
```

or 

```
Set-PnPPage "123.aspx" -Like:$false
```

Reply:
Hello @gautamdsheth, 

Thanks for the suggestion. I have changed accordingly now, Kindly review. 

Regards,
Nishkalank Bezawada